docs/install/update-harvester-configuration.md

```diff
@@ -47,6 +47,8 @@ If you upgrade from a version before `v1.1.2`, the `cloud-init` file in examples
 
 ### Configuration persistence
 
+The persistent filename previously was `/oem/99_custom.yaml`. It is now `/oem/90_custom.yaml` in Harvester v1.1.2 and later versions. That is because we use a newer `elemental` version. This file is created by `elemental` with installation time. If you upgrade from a version before `v1.1.2`, this file will be `/oem/99_custom.yaml`. We do not rename this file with an upgrade because we do not want the user to get confused. If you align the new mechanism, you can manually rename this file to `/oem/90_custom.yaml`.
```

Comment:
lgtm, just need to add to both v1.1. and v1.2 versions later

Reply:
OK. I will add both v1.1 and v1.2 when we completed the review